src/radiation.jl

```diff
@@ -0,0 +1,72 @@
+
+"""
+    blackbody_radiation(λ, T) -> radiance
+
+Accroding to Planck's law, calculate the spectral intensity of blackbody radiation
+at wavelength `λ` and temperature `T`.
+
+# Arguments
+- `λ` : Wavelength [m]
+- `T` : Temperature [K]
+
+# Return
+- `radiance` : Spectral radiance [W/m²/m/steradian]
+
+c.f. https://github.com/JuliaAstro/Planck.jl/blob/main/src/Planck.jl
```

Comment:
```suggestion
cf. https://github.com/JuliaAstro/Planck.jl/blob/main/src/Planck.jl
```
https://en.wikipedia.org/wiki/Cf.